stix_shifter_modules/sentinelone/README.md

```diff
@@ -0,0 +1,685 @@
+            "created": "2022-04-20T11:52:32.202Z",
+            "modified": "2022-04-20T11:52:32.202Z",
+            "objects": {
+                "0": {
+                    "type": "x-oca-event",
+                    "agent": "xx",
+                    "host_ref": "1",
+                    "created": "2022-04-20T05:31:03.739Z",
+                    "action": "Process Creation",
+                    "category": [
+                        "process"
+                    ],
+                    "process_ref": "2",
+                    "user_ref": "7"
+                },
+                "1": {
+                    "type": "x-oca-asset",
+                    "extensions": {
+                        "x-sentinelone-endpoint": {
+                            "agent_uuid": "xx",
+                            "machine_type": "xx",
+                            "endpoint_os": "windows"
+                        }
+                    },
+                    "hostname": "xx"
+                },
+                "2": {
+                    "type": "process",
+                    "command_line": "C:\\Program Files (x86)\\Google\\Temp\\GUM10C0.tmp\\xx /update /sessionid {A621AB5C-61F8-473C-91D7-6C91A9F50036}",
+                    "binary_ref": "5",
+                    "extensions": {
+                        "x-sentinelone-process": {
+                            "integrity_level": "SYSTEM",
+                            "process_unique_id": "xx",
+                            "publisher": "GOOGLE LLC",
+                            "signed_status": "signed",
+                            "story_line_id": "xx",
+                            "verified_status": "verified"
+                        }
+                    },
+                    "name": "xx",
+                    "parent_ref": "6",
+                    "pid": 228,
+                    "created": "2022-04-20T05:31:33.088Z",
+                    "creator_user_ref": "7"
+                },
+                "3": {
+                    "type": "directory",
+                    "path": "C:\\Program Files (x86)\\Google\\Temp\\xx"
+                },
+                "5": {
+                    "type": "file",
+                    "hashes": {
+                        "SHA-1": "c0a98fd8c74d031f54fda658a1c67d8886b5e076"
+                    }
+                },
+                "6": {
+                    "type": "process",
+                    "name": "xx",
+                    "created": "2022-04-20T05:31:31.715Z"
+                },
+                "7": {
+                    "type": "user-account",
+                    "user_id": "NT AUTHORITY\\xx"
+                }
+            },
+            "first_observed": "2022-04-20T05:31:42.446Z",
+            "last_observed": "2022-04-20T11:52:32.202Z",
+            "number_observed": 1
+      }
+   ]
+}
+```
+
+### Limitations
+- Default data retention in sentinelone deep visibility is 14 days.
+
+### References
+- [SentinelOne](#https://www.sentinelone.com/)
```

Comment:
remove `#` from the url. it breaks the url link.

Reply:
# is removed from url link.